pwiz_tools/Skyline/Controls/Graphs/DetectionToolbarProperties.cs

```diff
@@ -129,10 +129,10 @@ private void cmbTargetType_SelectedIndexChanged(object sender, EventArgs e)
         public void SetQValueTo(float qValue)
         {
             if (qValue == .01f)
-                rbQValue01.Select();
+                rbQValue01.Checked = true;

```

Comment:
Is this the critical change? It does seem wrong to use Select() instead of Checked = true. Not sure why that would pass sometimes and not others, though.

Reply:
The only way I was able to reproduce the timeout failure was to focus on another window while the test was running and that traced down to using Select() method. I cannot be 100% sure, but I ran this test in a loop for an hour without any exceptions.